test/cpp/test_tensor.cpp

```diff
@@ -119,16 +119,6 @@ TEST_F(TensorTest, TestSize) {
   });
 }
 
-TEST_F(TensorTest, TestRelu) {
```

Comment:
This codegen removes the `relu` function in the `tensor_methods`, so would it be okay to remove these tests here?

Reply:
it is oK